### PR TITLE
Improve Introduction Content for Docs

### PR DESCRIPTION
**Pull request by kirkbrauer**

### CODE REVIEW

**Comment by mangelajo** [comment]
Is it really draft? I'm all for merging if you are OK, and we can continue to iterate with less PR conflicts

**Comment by kirkbrauer** [comment]
@mangelajo I need to finish the content haha. Still need to finish the last page about the service and ensure consistency.

**Comment by kirkbrauer** [comment]
Ok, I just pushed a bunch of updates. Needs more diagrams, but I think it explains a lot more now.

**Comment by mangelajo** [comment]
Thank you very much, reviewing! :)

File: docs/source/introduction/drivers.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+kind: ExporterConfig
+endpoint: grpc.jumpstarter.example.com:443
+token: xxxxx
+export:
+  # a DUTLink interface to the DUT
+  dutlink:
+    type: jumpstarter_driver_dutlink.driver.Dutlink
+    config:
+      storage_device: "/dev/disk/by-id/usb-SanDisk_3.2_Gen_1_5B4C0AB025C0-0:0"
+```
```
Comment (mangelajo) [inline docs/source/introduction/drivers.md:37]: we can probably provide simpler examples, I'll follow up :)

**Review by mangelajo — APPROVED** [review]
It's a nice improvement much cleaner, let's keep iterating!!

File: docs/source/introduction/drivers.md (hunk 1 above)
Comment (kirkbrauer) [reply docs/source/introduction/drivers.md:37]: Yeah, I just grabbed the first example I could find. We should also not re-use the same examples and provide a bit of variety here.

**Pull request merged**